## Problem Statement - Normal Distribution

The mean salaries of Data Scientists working in Chennai, India is calculated to be 7,00,000 INR with a standard deviation of 90,000 INR. The random variable salary of Data Scientists follows a normal distribution.

  - What is the probability that a Data Scientist in Chennai has a salary more than 10,00,000 INR?
  - What is the probability that a Data Scientist in Chennai has a salary between 6,00,000 & 9,00,000 INR?
  - What is the probability that a Data Scientist in Chennai has a salary less than 4,00,000 INR?

In [16]:
import scipy.stats as stats

In [17]:
# Z-Score for x=10,00,000 for mean of 7,00,000 and sdev of of 90,000 INR
# z = (x-mean)/sdev

z=(1000000-700000)/90000

In [18]:
z

3.3333333333333335

we need to find the area of curve to the right of z value, for a z value of 3.33, which will be the probability that the salary will be more than 10,00,000 INR

In [19]:
1 - stats.norm.cdf(3.333)

0.00042957471189908336

- A cdf value of 0.0004 tells that there is a 0.4% probability that the salary will be more than 10,00,000 INR

In [20]:
# Z-Score for x=6,00,000 for mean of 7,00,000 and sdev of of 90,000 INR

z1= (600000-700000)/90000

# Z-Score for x=9,00,000 for mean of 7,00,000 and sdev of of 90,000 INR

z2= (900000-700000)/90000

In [21]:
z1, z2

(-1.1111111111111112, 2.2222222222222223)

- we need to find the difference in the area of curve to the left of z1=1.11 and the area of curve to the right of z2=2.22 value, which will be the probability that the salary will be between 6,00,000 & 9,00,000 INR

In [22]:
stats.norm.cdf(z2) - stats.norm.cdf(z1)

0.8536055914064735

- A cdf value of 0.853 tells that there is an 85.3% chance that the salary will be between 6,00,000 & 9,00,000 INR

In [23]:
# Z-Score for x=4,00,000 for mean of 7,00,000 and sdev of of 90,000 INR

z = (400000-700000)/90000

In [24]:
z

-3.3333333333333335

- we need to find the area of curve to the left of z value, for a z value of -3.33, which will be the probability that the salary will be less than 4,00,000 INR

In [27]:
stats.norm.cdf(-3.333)

0.00042957471189910326

- A cdf value of 0.369 tells that there is a 0.04% chance that the salary will be less than 4,00,000 INR